src/rollback_to_stable/rts.c

```diff
@@ -161,6 +161,11 @@ __wt_rts_btree_apply_all(WT_SESSION_IMPL *session, wt_timestamp_t rollback_times
           __wt_timestamp_to_string(rollback_timestamp, ts_string));
         WT_ERR(__wt_rts_history_final_pass(session, rollback_timestamp));
     }
+
+    /* Log a progress message for the history store. */
+    ++rollback_count;
+    __rts_progress_msg(session, rollback_timer, rollback_count, &rollback_msg_count);
```

Comment:
Won't this only log after we've finished with the history store?

Reply:
hmm true, I wonder if it's more beneficial to call the progress message after or before the history store? what do you think?

Reply:
If the goal is to have regular progress messages, ideally it'd be during RTS, kinda the same idea as what you've done in `__rts_btree_walk`.

Reply:
ok removed it! [245e6cf](https://github.com/wiredtiger/wiredtiger/pull/9194/commits/245e6cfa0efdda4e418b2f774ced4945332a22e3)